Resources/config/services.yml

```diff
@@ -0,0 +1,31 @@
+parameters:
+    sm.callback.class: "winzou\\Bundle\\StateMachineBundle\\Callback\\ContainerAwareCallback"
+
+services:
+    _defaults:
+        autowire:      true
```

Comment:
https://symfony.com/doc/current/service_container/autowiring.html#public-and-reusable-bundles